wdl-analysis/Cargo.toml

```diff
@@ -25,6 +26,7 @@ petgraph = { workspace = true }
 futures = { workspace = true }
 path-clean = { workspace = true }
 indexmap = { workspace = true }
+indicatif = { workspace = true }
```

Comment:
Similarly, this crate has been UX-agnostic until now, leaving the `wdl` crate (albeit locked away in the `bin` and not the `lib`) to implement and expose the UX.